mlflow/langchain/__init__.py

```diff
@@ -98,6 +98,7 @@ def save_model(
     metadata=None,
     loader_fn=None,
     persist_dir=None,
+    **kwargs,
```

Comment:
```suggestion
    example_no_conversion=False,
```


Can we add `example_no_conversion` as a keyword argument?